realm/src/androidTest/java/io/realm/TestHelper.java

```diff
@@ -46,6 +49,70 @@
 
 public class TestHelper {
 
+    /**
+     * Returns the corresponding column type for an object.
+     * @param o
+     * @return
+     */
+    public static ColumnType getColumnType(Object o){
+
+        if (o instanceof Boolean)
+            return ColumnType.BOOLEAN;
+        if (o instanceof String)
+            return ColumnType.STRING;
+        if (o instanceof Long)
+            return ColumnType.INTEGER;
+        if (o instanceof Float)
+            return ColumnType.FLOAT;
+        if (o instanceof Double)
+            return ColumnType.DOUBLE;
+        if (o instanceof Date)
+            return ColumnType.DATE;
+        if (o instanceof byte[])
+            return ColumnType.BINARY;
+
+        return ColumnType.MIXED;
+    }
+
+
+    /**
+     * Creates an empty table with 1 column of all our supported column types, currently 9 columns
+     * @return
+     */
+    public static Table getTableWithAllColumnTypes(){
+
+        Table t = new Table();
+
+        t.addColumn(ColumnType.BINARY, "binary");
```

Comment:
Why not do a 
```
for (ColumnType type : ColumnType.values()) {
  t.addColumn(type, type.toString().lowerCase();
}
``` 
instead? This will go out of date if/when we change the column types.